### PR TITLE
Fixed C#/VB samples

### PR DESCRIPTION
**Pull request by Potapy4**
Fixes dotnet/dotnet-api-docs#1833

### CODE REVIEW

**Comment by Potapy4** [comment]
FYI: There are also no examples for F# on this page (instead of the F# code, a fragment for C++ is displayed)

File: snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb [hunk 1]
```diff
@@ -2,9 +2,10 @@
 Option Strict On
 
 ' <Snippet2>
+Imports System
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb]: It's not necessary to import the System namespace; this Imports statement can be removed.

File: snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb [hunk 2]
```diff
@@ -2,9 +2,10 @@
...
 Imports System.Globalization
 
-Module Example
+Public Class Example
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb]: This code won't compile because it lacks an entry point. Please revert this change.

File: snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb [hunk 3]
```diff
@@ -16,10 +17,10 @@ Module Example
                            date1.ToString("D", culture))                 
       Next
    End Sub
-End Module
+End Class
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb]: Again, please revert this change.

File: snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern2.vb [hunk 4]
```diff
@@ -6,7 +6,7 @@ Option Strict On
 Imports System.Globalization
 Imports System.IO
 
-Module Example
+Public Class Example
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern2.vb]: Please revert lines 9 and 36

**Comment by rpetrusha** [comment]
Thank you for making the additional changes, @Potapy4. And again, thanks for addressing issue dotnet/dotnet-api-docs#1833. I'll merge your PR now.
